src/Core/src/Handlers/Editor/EditorHandler.cs

```diff
@@ -17,7 +17,8 @@ public partial class EditorHandler
 			[nameof(IEditor.TextColor)] = MapTextColor,
 			[nameof(IEditor.HorizontalTextAlignment)] = MapHorizontalTextAlignment,
 			[nameof(IEditor.VerticalTextAlignment)] = MapVerticalTextAlignment,
-			[nameof(IEditor.Keyboard)] = MapKeyboard
+			[nameof(IEditor.Keyboard)] = MapKeyboard,
+			[nameof(IEditor.CursorPosition)] = MapCursorPosition
```

Comment:
Should we also add the `SelectionLength` mappers here too?

Reply:
If I recall correctly, this has been done in another PR. Let me fetch upstream and see if it has already been added

Reply:
but we can do it on another PR also

Reply:
Nevermind, that was for Entry. I'll get it done 👍

Reply:
Added